### DISCUSSION TITLE
Export Bug

### DISCUSSION THREAD

**Discussion by razor950** [outline/outline#5943] [Self Hosting]
**To Reproduce**
1. Go to 'Export'
2. Click on 'Export Data'
3. Pick Markdown with attachments included.

**Expected behavior**
I expected to get a export of my outline in markdown format.

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Outline (please complete the following information):**
- Install: self-hosted
- Version: 0.72.0

It doesn't matter if I use different browsers or what not.

 Error: "Failed • Unexpected close tag Line: 5 Column: 7 Char: >"
 
 This is the full log for the error
 {"error":"Unexpected close tag\nLine: 5\nColumn: 7\nChar: >","level":"error","message":"Error processing task in ExportMarkdownZipTask","stack":"XMLParserError: Unexpected close tag\nLine: 5\nColumn: 7\nChar: >\n    at error (/opt/outline/node_modules/xml2js/node_modules/sax/lib/sax.js:651:10)\n    at strictFail (/opt/outline/node_modules/xml2js/node_modules/sax/lib/sax.js:677:7)\n    at closeTag (/opt/outline/node_modules/xml2js/node_modules/sax/lib/sax.js:871:9)\n    at SAXParser.write (/opt/outline/node_modules/xml2js/node_modules/sax/lib/sax.js:1436:13)\n    at exports.Parser.Parser.parseString (/opt/outline/node_modules/xml2js/lib/parser.js:327:31)\n    at Parser.parseString (/opt/outline/node_modules/xml2js/lib/parser.js:5:59)\n    at NodeXmlParser.parse (/opt/outline/node_modules/aws-sdk/lib/xml/node_parser.js:31:10)\n    at Request.extractError (/opt/outline/node_modules/aws-sdk/lib/services/s3.js:697:39)\n    at Request.callListeners (/opt/outline/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\n    at Request.emit (/opt/outline/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/opt/outline/node_modules/aws-sdk/lib/request.js:686:14)\n    at Request.transition (/opt/outline/node_modules/aws-sdk/lib/request.js:22:10)\n    at AcceptorStateMachine.runTo (/opt/outline/node_modules/aws-sdk/lib/state_machine.js:14:12)\n    at /opt/outline/node_modules/aws-sdk/lib/state_machine.js:26:10\n    at Request.<anonymous> (/opt/outline/node_modules/aws-sdk/lib/request.js:38:9)\n    at Request.<anonymous> (/opt/outline/node_modules/aws-sdk/lib/request.js:688:12)\n    at Request.callListeners (/opt/outline/node_modules/aws-sdk/lib/sequential_executor.js:116:18)\n    at Request.emit (/opt/outline/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/opt/outline/node_modules/aws-sdk/lib/request.js:686:14)\n    at Request.transition (/opt/outline/node_modules/aws-sdk/lib/request.js:22:10)\n    at AcceptorStateMachine.runTo (/opt/outline/node_modules/aws-sdk/lib/state_machine.js:14:12)\n    at /opt/outline/node_modules/aws-sdk/lib/state_machine.js:26:10"}

**Comment by razor950** [comment]
Follow-up forgot to mention if I don't include attachments, it exports fine via markdown.

**Reply by tommoor** [reply]
The server cannot access your file storage to download the attachments, this is usually a DNS resolution or firewall issue

**Reply by razor950** [reply]
wouldn't that mean the attachments wouldn't show up in outline at all?

**Reply by tommoor** [reply]
Not neccessarily, when exporting the request is made from the server itself rather than the browser
